web3/_utils/abi.py

```diff
@@ -385,6 +386,7 @@ def check_if_arguments_can_be_encoded(
     )
 
 
+@deprecated_for("eth_utils.abi.get_normalized_abi_inputs")
```

Comment:
Again, do we need to deprecate in `v7`? Now might be the time to break it and deprecate in `v6`. I would argue anything living under `_utils` does _not_ need a deprecation message in either case, so this is probably fine to just break in `v7` and not even deprecate in `v6`.

Reply:
Agreed, deprecations should be made in `v6`. Trying to keep track of all the things, deprecating this private method for now but I do intend to remove it in this PR.